.github/workflows/e2e.yaml

```diff
@@ -67,8 +70,8 @@ jobs:
 
       - name: Run e2e tests
         env:
-          IMAGE_NAME: ghcr.io/${LOWERCASE_REPO_OWNER}/container-toolkit
-          VERSION: ${COMMIT_SHORT_SHA}
+          IMAGE_NAME: ghcr.io/nvidia/container-toolkit
```

Comment:
If we do move `e2e.yaml` to a central repo, we would have to have this as an input.

(Out of scope for this PR).

Reply:
agree